src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -250,6 +252,22 @@ public void testPullPrivateRepoWithoutAuth() throws Exception {
   }
 
   @Test
+  public void testBuildImageIdWithBuildargs() throws Exception {
```

Comment:
In the Dockerfile, I have added a bash line to throw exception if buildarg is not set. I can add tests for other parameters.